P202.Training.WCF/Handlers/CreateUserHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Agatha.Common;
+using Agatha.ServiceLayer;
+using P202.Training.Data.Repositories;
+using P202.Training.Domain;
+using P202.Training.WCF.RequestsAndResponses;
+
+namespace P202.Training.WCF.Handlers
+{
+    public class CreateUserHandler : RequestHandler<CreateUserRequest, CreateUserResponse>
+    {
+        private readonly IUsersService _usersService;
+        private readonly IUserRepository _userRepository;
```

Comment:
API layer shouldn't have a reference to the repository. Repositories should be accessed only from the Domain layer